From: Patrik Fimml <patrik@fimml.at>
Subject: [PATCH] fixes/link-plugins-with-cxx

Make automake think plugins are C++ targets, so it links them properly.

Signed-off-by: Patrik Fimml <patrik@fimml.at>

---
 plugins/aiksaurus/Makefile.am |    1 +
 plugins/command/Makefile.am   |    1 +
 plugins/gda/Makefile.am       |    1 +
 plugins/gdict/Makefile.am     |    1 +
 plugins/goffice/Makefile.am   |    1 +
 plugins/ots/Makefile.am       |    1 +
 plugins/psion/Makefile.am     |    1 +
 7 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/plugins/aiksaurus/Makefile.am b/plugins/aiksaurus/Makefile.am
index 892c1a7..77fa8b6 100644
--- a/plugins/aiksaurus/Makefile.am
+++ b/plugins/aiksaurus/Makefile.am
@@ -22,4 +22,5 @@ aiksaurus_la_LDFLAGS = \
 	-no-undefined
 
 aiksaurus_la_SOURCES = 
+nodist_EXTRA_aiksaurus_la_SOURCES = dummy.cpp
 
diff --git a/plugins/command/Makefile.am b/plugins/command/Makefile.am
index 0f22f0e..4315bb9 100644
--- a/plugins/command/Makefile.am
+++ b/plugins/command/Makefile.am
@@ -17,5 +17,6 @@ command_la_LDFLAGS = \
 	-no-undefined
 
 command_la_SOURCES = 
+nodist_EXTRA_command_la_SOURCES = dummy.cpp
 
 endif
diff --git a/plugins/gda/Makefile.am b/plugins/gda/Makefile.am
index c84c7fc..965191a 100644
--- a/plugins/gda/Makefile.am
+++ b/plugins/gda/Makefile.am
@@ -15,6 +15,7 @@ gda_la_LDFLAGS = \
 	-no-undefined
 
 gda_la_SOURCES = 
+nodist_EXTRA_gda_la_SOURCES = dummy.cpp
 
 EXTRA_DIST = \
 	plugin.m4
diff --git a/plugins/gdict/Makefile.am b/plugins/gdict/Makefile.am
index 26dc68f..e96edff 100644
--- a/plugins/gdict/Makefile.am
+++ b/plugins/gdict/Makefile.am
@@ -22,6 +22,7 @@ gdict_la_LDFLAGS = \
 	-no-undefined
 
 gdict_la_SOURCES = 
+nodist_EXTRA_gdict_la_SOURCES = dummy.cpp
 
 EXTRA_DIST = \
 	plugin.m4
diff --git a/plugins/goffice/Makefile.am b/plugins/goffice/Makefile.am
index 4ec7f7b..8dd92ad 100644
--- a/plugins/goffice/Makefile.am
+++ b/plugins/goffice/Makefile.am
@@ -15,6 +15,7 @@ goffice_la_LDFLAGS = \
 	-no-undefined
 
 goffice_la_SOURCES = 
+nodist_EXTRA_goffice_la_SOURCES = dummy.cpp
 
 EXTRA_DIST = \
 	plugin.m4
diff --git a/plugins/ots/Makefile.am b/plugins/ots/Makefile.am
index 1d9c438..3754978 100644
--- a/plugins/ots/Makefile.am
+++ b/plugins/ots/Makefile.am
@@ -15,6 +15,7 @@ ots_la_LDFLAGS = \
 	-no-undefined
 
 ots_la_SOURCES = 
+nodist_EXTRA_ots_la_SOURCES = dummy.cpp
 
 EXTRA_DIST = \
 	plugin.m4
diff --git a/plugins/psion/Makefile.am b/plugins/psion/Makefile.am
index a216aba..3db02ac 100644
--- a/plugins/psion/Makefile.am
+++ b/plugins/psion/Makefile.am
@@ -15,6 +15,7 @@ psion_la_LDFLAGS = \
 	-no-undefined
 
 psion_la_SOURCES = 
+nodist_EXTRA_psion_la_SOURCES = dummy.cpp
 
 EXTRA_DIST = \
 	plugin.m4
-- 
tg: (a6c4321..) fixes/link-plugins-with-cxx (depends on: master fixes/fully-link-plugins)
